redbot/cogs/audio/core/commands/llset.py

```diff
@@ -188,31 +188,6 @@ async def command_llsetup_password(self, ctx: commands.Context, password: str):
                 ),
             )
 
-    @command_llsetup.command(name="restport")
```

Comment:
Please take a look at this a little closer. I have a detailed traceback I will pass to you, but setting the wsport to something other than 2333 on an internal server causes a traceback when trying to connect with play or other audio commands.

Reply:
Should be resolved, issue appeart to originate from having old instance of nodes in the node pool of RLL, because `llset` commands didnt do cleanup here (nor does RLL). I added some cleanup and havent been able to  reproduce.